packages/dappmanager/test/integration/migrationCompose/migration.test.int.ts

```
@@ -0,0 +1,161 @@
+    const composeExpected = `
+version: '3.4'
+networks:
+  dncore_network:
+    external: true
+    name: dncore_network
+services:
+  dappmanager.dnp.dappnode.eth:
+    image: chentex/random-logger
+    container_name: DAppNodeCore-dappmanager.dnp.dappnode.eth
+    restart: always
+    dns: 172.33.1.2
+    networks:
+      dncore_network:
+        ipv4_address: 172.33.1.7`;
+
+    migrateCoreNetworkInCompose(container);
+    const composeAfter = fs.readFileSync(
+      `${testMigrationPath}/test-migration/docker-compose.yml`,
+      { encoding: "utf8" }
+    );
+    expect(composeAfter.trim()).to.equal(composeExpected.trim());
+  });
+
+  it("Should do alias migration", () => {
+    const composeExpected = `
+version: '3.4'
+networks:
+  dncore_network:
+    external: true
+    name: dncore_network
+services:
+  dappmanager.dnp.dappnode.eth:
+    image: chentex/random-logger
+    container_name: DAppNodeCore-dappmanager.dnp.dappnode.eth
+    restart: always
+    dns: 172.33.1.2
+    networks:
+      dncore_network:
+        ipv4_address: 172.33.1.7
+        aliases:
+          - dappmanager.dnp.dappnode.eth.test-migration.dappnode`;
+
+    const alias = getPrivateNetworkAlias(container);
+    addNetworkAliasCompose(container, dncoreNetwork, [alias]);
+    const composeAfter = fs.readFileSync(
+      `${testMigrationPath}/test-migration/docker-compose.yml`,
+      { encoding: "utf8" }
+    );
+    expect(composeAfter.trim()).to.equal(composeExpected.trim());
+  });
+
+  after("Remove test setup", async () => {
+    // Disconnect from network
+    await shell(
+      `docker network disconnect ${dncoreNetwork} ${containerName} --force`
+    );
+    // Remove network
+    await shell(`docker network rm dncore_network`);
+    // Remove container
+    await shell(`docker rm ${containerName} --force`);
+    // Remove image
+    await shell(`docker image rm ${randomImage}`);
+    // Remove dir
+    await shell(`rm -rf ${testMigrationPath}/test-migration`);
```

Use shellSafe on all of this. If the test fails early those may not be created